src/Task/PhpCsFixerV2.php

```diff
@@ -91,6 +86,8 @@ public function run(ContextInterface $context)
             return $this->runOnAllFiles($context, $arguments);
         }
 
+        $arguments->add('--path-mode=intersection');
```

Comment:
poor, quick and dirty solution would be to execute command with `intersection` mode, if it crashes on not fully defined finder, run it with `override` mode...
or, as suggested somewhere, execute `Runner` programatically instead of calling CLI command.